src/centralserver/admin-ui/src/components/layout/MainContentArea.vue

```diff
@@ -0,0 +1,51 @@
+<!--
+   The MIT License
+   Copyright (c) 2019- Nordic Institute for Interoperability Solutions (NIIS)
+   Copyright (c) 2018 Estonian Information System Authority (RIA),
+   Nordic Institute for Interoperability Solutions (NIIS), Population Register Centre (VRK)
+   Copyright (c) 2015-2017 Estonian Information System Authority (RIA), Population Register Centre (VRK)
+
+   Permission is hereby granted, free of charge, to any person obtaining a copy
+   of this software and associated documentation files (the "Software"), to deal
+   in the Software without restriction, including without limitation the rights
+   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+   copies of the Software, and to permit persons to whom the Software is
+   furnished to do so, subject to the following conditions:
+
+   The above copyright notice and this permission notice shall be included in
+   all copies or substantial portions of the Software.
+
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+   THE SOFTWARE.
+ -->
+<template>
+  <div class="xrd-view-common content">
+    <slot></slot>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+
+export default Vue.extend({});
+</script>
+
+<style scoped lang="scss">
+.content {
+  margin-top: 40px;
+  padding-left: 10px;
+  padding-right: 10px;
+  width: 100%;
+  max-width: 1600px;
+
+  padding-left: 92px;
+  @media only screen and (max-width: 920px) {
```

Comment:
Should the breakpoints (e.g. `920px`) be separated to a theme file of sorts since they are used in multiple files. Also the colors and maybe even space widths could be gathered into one theme that would be easy to maintain.

Just an idea. What do you think, is it too big of a change? Because we don't use a theme in Security Server either

Reply:
**X-Road global theme**: 
I think it would be a good idea to have common themefiles for centralserver adminview and proxy-api-ui but I'm afraid doing that refactoring would take quite some time. This should have been though before/during proxy-api-ui implementation, now it's laborous. I think we should create a separate ticket to see whether that refactoring would be doable in reasonable time. 

We already have a couple of files containing global styles in centralserver admin view. We could try to approach 'global theme' by splitting those files to a somekind of theme. But that could be done between mock-view tickets to avoid confusion. 

**CSS-toolkits**: 
Vuetify also contains some helper classes, grid-system etc. I think using those helper kits would introduce some duplication there. Also that kind of tools are not used in proxy-api-ui so I'd prefer using the same approach in centralserver adminui.

**Media styles to global styles**: 
I guess moving styles in this component to global stylefiles would make sense in many cases but I designed this maincontentarea -component so that it shouldn't be necessary to move media styles global (of course this could be refactored if this component is too restrictive). If this component can be applied to all the pages where 1600px max-width view is used then I don't see why these styles be made global. 

MainContentArea -component could be moved to shared-ui also.

Reply:
Yes, Vuetify already seems to have quite a vast selection of utility classes. Maybe they should just be utilised more instead of creating duplicates. But that is not related to this task what so ever.

Also about the themes, maybe the Vuetify theme could be customised so there would be no need for creating one from scratch https://vuetifyjs.com/en/features/theme/#customizing
But that probably also requires some effort.